.eslintignore

```diff
@@ -1,6 +1,6 @@
 **/node_modules
-**/coverage
-test/fixtures
+**/.coverage
 build/
 docs/
 protos/
+system-test/
```

Comment:
I am a little confused.  Are these hand written changes over top of the generated ignorefile?  This should be overwritten by synth:
https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/node_library/.eslintignore

I think we *want* to lint the `system-test` directory.

Reply:
These are all autogeneraed code (I removed the hacks in synth.py about skipping `eslintignore`)
I thought it's not needed any more, so I tried to remove all hacks and see if it works.
I will try upgrade synthtool and run it again, it should be overwritten by synth properly.

Reply:
Reran synthtool and it's changed back by gts. Thanks!